### ISSUE TITLE
fatal errorL libxmlparser.h espa_metadata.h

### ISSUE THREAD

**Issue by abanskota** [USGS-EROS/espa-product-formatter#29]
I have spent over a week trying to install espa_product_formatter without success. The error message starts with "espa_metadata.h:20;27: fatal error: libxml/parser.h: No such file or directory" followed by whole bunch of no such files for espa_metadata.h and other espa related header files. I have diligently followed the instructions on setting environment variables related to include and library directories  as well as ESPAINC, ESPALIB, PREFIX, and ESPA_LAND_MASS_POLYGON. I tried both 'sudo apt-get install' as well as building from source for all the dependent libraries. I wonder what might have gone wrong.

**Comment by gschmidt-usgs** [comment]
Does the directory which you set your XML2INC environment variable to
contain a 'libxml' subdirectory?  You may need to reset you XML2INC
environment variable so that it is the level above the libxml
subdirectory.  Then try rebuilding.

-----------------------------------------------------------------------------------------------------
Gail Schmidt
Software Engineer
Stinger Ghaffarian Technologies (SGT)
Contractor to the U.S. Geological Survey (USGS)
Earth Resources Observation and Science (EROS) Center
47914 252nd Street
Sioux Falls, SD 57198
Phone: 605-593-2272
gail.schmidt.ctr@usgs.gov

NOTICE:  This email may contain confidential, proprietary, or competition
sensitive bid or proposal procurement information.  Unauthorized disclosure
of this information may carry criminal penalties as set forth in the
Procurement Integrity Act, 41 U.S.C. 423, as amended.  Further, the
unauthorized disclosure of certain commercial information by civil servants
may result in fines or imprisonment under the Trade Secrets Act (18 U.S.C.
1905).  If you have received this information in error, please delete it,
including all copies, and notify the sender of the error immediately.

**Comment by abanskota** [comment]
Yes, it is set one level above the libxml directory i.e. /user/include/libxml2. The libxml2 directory contains the libxml subdirectory.

**Comment by gschmidt-usgs** [comment]
And, does that /user/include/libxml2/libxml/include directory contain the
"parser.h" file?

-----------------------------------------------------------------------------------------------------
Gail Schmidt
Software Engineer
Stinger Ghaffarian Technologies (SGT)
Contractor to the U.S. Geological Survey (USGS)
Earth Resources Observation and Science (EROS) Center
47914 252nd Street
Sioux Falls, SD 57198
Phone: 605-593-2272
gail.schmidt.ctr@usgs.gov

NOTICE:  This email may contain confidential, proprietary, or competition
sensitive bid or proposal procurement information.  Unauthorized disclosure
of this information may carry criminal penalties as set forth in the
Procurement Integrity Act, 41 U.S.C. 423, as amended.  Further, the
unauthorized disclosure of certain commercial information by civil servants
may result in fines or imprisonment under the Trade Secrets Act (18 U.S.C.
1905).  If you have received this information in error, please delete it,
including all copies, and notify the sender of the error immediately.

**Comment by abanskota** [comment]
Yes, it does.

**Comment by gschmidt-usgs** [comment]
Can you send me the stdout for the build information that is failing?

-----------------------------------------------------------------------------------------------------
Gail Schmidt
Software Engineer
Stinger Ghaffarian Technologies (SGT)
Contractor to the U.S. Geological Survey (USGS)
Earth Resources Observation and Science (EROS) Center
47914 252nd Street
Sioux Falls, SD 57198
Phone: 605-593-2272
gail.schmidt.ctr@usgs.gov

NOTICE:  This email may contain confidential, proprietary, or competition
sensitive bid or proposal procurement information.  Unauthorized disclosure
of this information may carry criminal penalties as set forth in the
Procurement Integrity Act, 41 U.S.C. 423, as amended.  Further, the
unauthorized disclosure of certain commercial information by civil servants
may result in fines or imprisonment under the Trade Secrets Act (18 U.S.C.
1905).  If you have received this information in error, please delete it,
including all copies, and notify the sender of the error immediately.

**Comment by abanskota** [comment]
Actually, the problem got resolved. The problem was related to environmental variables not being visible with 'Sudo make'. http://unix.stackexchange.com/questions/19983/when-running-sudo-make-install-environment-variables-are-not-passed. 
I used Sudo -E env make', and it worked. Thanks for your prompt response.
